app/models/assignment.js

```diff
@@ -153,6 +153,20 @@ export default class AssignmentModel extends Model {
     return dcpPlannedcompletiondate;
   }
 
+  @computed('tab', 'dcpLupteammemberrole', 'project.milestones')
+  get toReviewMilestoneTimeRemaining() {
+    const participantMilestoneId = this.milestoneConstants.referralIdentifierByAcronymLookup[this.dcpLupteammemberrole];
+    const { dcpRemainingplanneddayscalculated } = this.project.get('milestones').find(milestone => milestone.dcpMilestone === participantMilestoneId) || {};
+    return dcpRemainingplanneddayscalculated;
+  }
+
+  @computed('tab', 'dcpLupteammemberrole', 'project.milestones')
+  get toReviewMilestoneTimeDuration() {
+    const participantMilestoneId = this.milestoneConstants.referralIdentifierByAcronymLookup[this.dcpLupteammemberrole];
+    const { dcpActualdurationasoftoday } = this.project.get('milestones').find(milestone => milestone.dcpMilestone === participantMilestoneId) || {};
+    return dcpActualdurationasoftoday;
+  }
```

Comment:
Makes sense, good choice. This computed depends on properties from assignment and project.